src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputNonEmptyAtclauseDescriptionCheck.java

```diff
@@ -7,7 +7,6 @@
 	 * @param a Some javadoc
 	 * @param b Some javadoc
 	 * @param c Some javadoc
-	 * @return Some javadoc
```

Comment:
Why do inputs need to change?  Nothing hurts from leaving them in since check won't process them.
Plrease revert this change. This will fix the changes to line numbers too in the test.